README.md

```diff
@@ -15,24 +15,25 @@ pest is a [PEG](https://en.wikipedia.org/wiki/Parsing_expression_grammar) parser
 
 ## Elegant grammar
 
-Defining a grammar for an alpha-numeric identifier that does not start with a digit is a straight-forward as:
+Defining a grammar for a list of alpha-numeric identifiers that do not start with a digit is a straight-forward as:
 
-```rust
+```
```

Comment:
I would leave the `rust` tag here in order to have *some* highlighting, even though it's not technically correct.